setup.md

```diff
@@ -86,12 +107,10 @@ sudo docker run \
     --name orca_dev \
     -ti \
     -e JUPYTER_ENABLE_LAB=yes \
-    -v ~/OrcaDetector/orca_detector:/src \
```

Comment:
Per my comments in the Dockerfile, during development, I have a pretty strong preference for volume mapping so we can manage things like our development process and git commits outside of the container, rather than freezing all of these things into a Docker image.

Reply:
Agreed, I'm reverting this now.

Reply:
Reverted. Please notice that I slightly changed the folders mapping. I wanted to map the source on the root repo level to be able to commit from inside the container.
Also, I kept the symmetry on the weights folder so I can do development from my local file system too

Reply:
acknowledged